feat: .net9 and ge6

gerhardol opened the pull request:
GitExtensions.Extensibility 0.4.0

untested, builds only locally

gerhardol committed:
feat: .net9 and ge6

Update `GitExtensions.Extensibility` version in GitExtensions.AzureDevOpsCommitMessage.csproj

Co-authored-by: Marek Fišera <mara@neptuo.com>

maraf commented on src/GitExtensions.AzureDevOpsCommitMessage/GitExtensions.AzureDevOpsCommitMessage.csproj:
```diff
@@ -29,10 +29,10 @@
-    <PackageReference Include="Microsoft.VisualStudio.Threading" Version="17.4.27" />
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All"/>
+    <PackageReference Include="GitExtensions.Extensibility" Version="0.4.0" />
```
```suggestion
    <PackageReference Include="GitExtensions.Extensibility" Version="0.4.0.*" />
```

pmiossec committed:
ci: hardcode version to download

until https://github.com/gitextensions/gitextensions.extensibility/issues/36 is fixed
